pytorch_lightning/loggers/wandb.py

```diff
@@ -51,11 +51,8 @@ class WandbLogger(LightningLoggerBase):
         anonymous: Enables or explicitly disables anonymous logging.
         version: Sets the version, mainly used to resume a previous run.
         project: The name of the project to which this run will belong.
-        tags: Tags associated with this run.
         log_model: Save checkpoints in wandb dir to upload on W&B servers.
         experiment: WandB experiment object
-        entity: The team posting this run (default: your username or your default team)
-        group: A unique string shared by all runs in a given group
```

Comment:
We should probably describe that additional keywords (mention group, tag ect by name) used by wanddb init can be feed in as keywords when initializing the logger

Reply:
is it good now??

Reply:
Very good :]